Carbon.IIS/Functions/Set-CIisCollection.ps1

```diff
@@ -0,0 +1,193 @@
+function Set-CIisCollection
+{
+    <#
+    .SYNOPSIS
+    Sets the exact contents of an IIS configuration section collection.
+
+    .DESCRIPTION
+    The `Set-CIisCollection` function sets IIS configuration collection to a specific set of items. Pipe the collection
+    items to the function (or pass them to the `InputObject` parameter). You can pass just item vales if the collection
+    items only have single values. Otherwise, pass a hashtable of name/value pairs.
+
+    To make changes to a global configuration section, pass its path to the `SectionPath` parameter. To make changes to
+    a site, directory, application, or virtual directory, pass its pass location path to the `LocatinPath`. To make

```

Comment:
I would rephrase this because it sounds like you would use either `SectionPath` or `LocationPath` exclusively, when really you use `LocationPath` in addition to `SectionPath`.